pkg/rancher-desktop/utils/protocols.ts

```diff
@@ -77,19 +36,24 @@ export const protocolsRegistered = Latch();
  * production environments.
  */
 function setupAppProtocolHandler() {
-  // TODO: #5659 - Deprecated in Electron 25; replace with protocol.handle
-  // eslint-disable-next-line deprecation/deprecation
-  const registrationProtocol = isDevBuild ? protocol.registerHttpProtocol : protocol.registerFileProtocol;
-
-  // TODO: #5659 - Deprecated in Electron 25; replace with protocol.handle
-  // eslint-disable-next-line deprecation/deprecation
-  registrationProtocol('app', (request, callback) => {
-    const relPath = decodeURI(new URL(request.url).pathname);
-    const redirectUrl = redirectedUrl(relPath);
-    const result = getProtocolResponse(request, redirectUrl, relPath);
-
-    callback(result);
-  });
+  protocol.handle(
+    'app',
+    (request) => {
+      const relPath = decodeURI(new URL(request.url).pathname);
```

Comment:
Wouldn't `(url).pathname` already be decoded? Why do we need to `decodeURI` that again? Though I see we did that before too…

Reply:
iirc, it might have had to do with path names on Windows, but I'll have to do some testing to see if my memory serves me correctly..